### ISSUE TITLE
Better background on using registries

### ISSUE THREAD

**Issue by jdolitsky** [helm/helm-www#637]
see https://github.com/helm/helm-www/pull/623#discussion_r422184389

**Comment by phroggyy** [comment]
One thing I realised while talking to @hickeyma was that the docs I submitted in #623 are a bit off as the example used there is `nginx:1.10`, this won't work since nginx is not a helm Chart, so we could update any references I've made to nginx to match the existing registry docs with `localhost:5000/myrepo/mychart:2.7.0`.

Commenting here just to have a record of it.

**Comment by hickeyma** [comment]
Yeah, I think the docs could do with improvement especially for people new to registries.
